readme/as-IN.md

```
@@ -101,42 +101,42 @@ export default async function (): Promise<NextConfig> {
 ---
 
-### এই ৰিপ'ৰ ভিতৰত কি আছে?
+### এই repo ৰ ভিতৰত কি আছে?
 
-| সঁজুলি       | সংক্ষিপ্ত বিৱৰণ                                                                       | ডকুমেণ্ট                                |
-| ------------ | ------------------------------------------------------------------------------------- | --------------------------------------- |
-| **Compiler** | বিল্ড-টাইম React স্থানীয়কৰণ                                                          | [/compiler](https://lingo.dev/compiler) |
-| **CLI**      | ৱেব আৰু মোবাইল এপ্, JSON, YAML, markdown, + অধিকৰ বাবে এক-কমাণ্ড স্থানীয়কৰণ          | [/cli](https://lingo.dev/cli)           |
-| **CI/CD**    | প্ৰতিটো পুছত অনুবাদ স্বয়ংক্ৰিয়ভাৱে কমিট কৰক + প্ৰয়োজন হ'লে pull request সৃষ্টি কৰক | [/ci](https://lingo.dev/ci)             |
-| **SDK**      | ব্যৱহাৰকাৰী-উৎপন্ন সমলৰ বাবে ৰিয়েলটাইম অনুবাদ                                        | [/sdk](https://lingo.dev/sdk)           |
+| Tool         | TL;DR                                                                     | Docs                                    |
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe has extra space to the left for style "compact"

(MD060, table-column-style)